2_adventures/001/chitankensuru/04_gosu_tictactoe/README.md

```diff
@@ -0,0 +1,8 @@
+#Gosu try
+
+Just following this guy's video tutorial to make tic tac toe in ruby with gosu
+https://www.youtube.com/watch?v=tQjjHt0I3_o
+
+Extra things learned today
+.git/info/exclude works similar like .gitignore except it won' be part of a
```

Comment:
Funny, I was reading the same thing yesterday, with "4 Ways to Ignore/Exclude Files in Git Repositories".